utils/prezly/api/PrezlyApi.ts

```diff
@@ -9,50 +10,85 @@ type SortOrder = 'desc' | 'asc';
 export default class PrezlyApi {
     private readonly sdk: PrezlySDK;
 
-    constructor(accessToken: string) {
+    private readonly newsroomUuid: Newsroom['uuid'];
+
+    private newsroom?: Newsroom;
+
+    constructor(accessToken: string, newsroomUuid: Newsroom['uuid']) {
         this.sdk = new PrezlySDK({ accessToken });
+        this.newsroomUuid = newsroomUuid;
+        this.sdk.newsrooms.get(this.newsroomUuid).then((nr) => this.newsroom = nr);
```

Comment:
Hmm, is `nr` a `number`? 🙂 

---

I don't think that we should do this kind of async operation in constructor. Assume that somewhere in code you will have:
```
const api = new PrezlyApi(...); // this makes a request to fetch a newsroom (line 20)
api.getNewsroom(...); // this will immediately make another request (line 29), because first one is still pending
```